packages/manager/apps/pci-block-storage/src/pages/new/components/AvailabilityZoneStep.tsx

```diff
@@ -0,0 +1,69 @@
+import { TilesInputComponent } from '@ovh-ux/manager-react-components';
+import { useState } from 'react';
+import { OsdsButton, OsdsText } from '@ovhcloud/ods-components/react';
+import { ODS_BUTTON_SIZE } from '@ovhcloud/ods-components';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import { Step } from '@/pages/new/hooks/useStep';
+import { TRegion } from '@/api/data/regions';
+
+interface AvailabilityZoneStepProps {
+  region: TRegion;
+  step: Step;
+  onSubmit: (zone: string) => void;
+}
+
+export function AvailabilityZoneStep({
+  region,
+  step,
+  onSubmit,
+}: Readonly<AvailabilityZoneStepProps>) {
+  const { t } = useTranslation('stepper');
+
+  const [selectedZone, setSelectedZone] = useState<string | undefined>(
```

Comment:
If `TilesInputComponent` component accepts `null` as `value`, prefer instantiate it with `null` over `undefined`